rocky/account/views/login.py

```diff
@@ -51,7 +51,7 @@ def get_form(self, step=None, **kwargs):
     def get_context_data(self, form, **kwargs):
         context = super().get_context_data(form, **kwargs)
         # if default device is set then the user has enabled two factor auth
-        context["two_factor_enabled"] = default_device(self.request.user)
+        context["two_factor_enabled"] = settings.TWOFACTOR_ENABLED or default_device(self.request.user)
```

Comment:
Wouldn't just assigning this to `settings.TWOFACTOR_ENABLED` be enough?

Reply:
When the 2 factor is enabled is, we also want to check if the 2-auth device is set.

Reply:
Then you should use the `and` operator and not `or` right?

Reply:
Both would be wrong. The problem is that we don't support settings.TWOFACTOR_ENABLED in the login template. Putting "Two factor authentication is enabled for your account" instead of ""Two factor authentication is not enabled for your account. Enable it to continue" on the page when two factor is completely disabled isn't the correct way to fix this.